addons/l10n_pe_pos/views/templates.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<odoo>
+
+    <template id="ticket_validation_screen" inherit_id="point_of_sale.ticket_validation_screen">
+        <!-- We do not let the user create the invoice by themselves. That is why we hide the
+        "Get my Invoice" button -->
+        <xpath expr="//div[@id='get_my_invoice']" position="attributes">
+            <attribute name="t-if">pos_order.company_id.country_code != 'PE'</attribute>
```

Comment:
You should add a condition term, in case there are several localizations installed, similar as you did below:

```suggestion
            <attribute name="t-if" add="(pos_order.company_id.country_code != 'PE')" separator=" and " />
```